capitalize.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.capitalize
```

Comment:
- [ ] Also it should be corrected before capitalization. hint: `@neamable.correct_name.capaitalize`